server/src/main/java/org/opensearch/index/store/remote/utils/cache/CacheUsage.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.store.remote.utils.cache;
```

Comment:
@nknize What do you think about the location for this code? I kind of like keeping it contained here until either it matures a bit or we replace it with something else. But marking it experimental in the `opensearch-commons` library essentially accomplishes the same thing, so I'm happy to defer to your preference.

Reply:
I put my thoughts in the comments below..